config/config-package.py

```diff
@@ -273,7 +273,9 @@ def pyproject_toml(self):
         changes_extension = 'rst'
         if (self.path / 'CHANGES.md').exists():
             changes_extension = 'md'
-            destination = self.path / 'news' / '.changelog_template.jinja'
+            news = self.path / 'news'
+            news.mkdir(parents=True, exist_ok=True)
```

Comment:
Indeed, thanks! In all projects that I applied `plone/meta` it already had a `news folder...

We might want to add a `.gitkeep` file, otherwise the folder will be deleted when `towncrier` updates the `CHANGES` file and removes the old news entries.

Reply:
👍🏼 done